codebase/cli.py

```diff
@@ -0,0 +1,55 @@
+import re
+import sys
+help = """
+CSV : summarized csv file
+
+USAGE: python cli.py [OPTIONS]
+
+OPTIONS:
+-e --eg start−up example = nothing
+-d --dump on test failure, exit with stack dump = false
+-f --file file with csv data = ../data/auto93.csv
+-h --help show help = false
+-n --nums number of nums to keep = 512
+-s --seed random number seed = 10019
+-S --seperator feild seperator = ,""" 
+
+
+def coerce(s):
+    def fun(s1):
+        if s1 == "true" or s1 == "True":
+            return True
+        if s1 == "false" or s1 == "False":
+            return False
+        return s1
+    if s.isdigit():
```

Comment:
We need to check for float values also, not just integers.

Reply:
![image](https://user-images.githubusercontent.com/55804665/188637536-93cb2499-0a20-4fad-9bd1-bc18d8c057a1.png)
We will only get integers as input for seed and nums